qna/DE-was-macht-ft.md

```diff
@@ -0,0 +1,39 @@
+## Question
+
+Welche Produkte bietet fiskaltrust für wen an?
+
+## Question
+
+Was macht fiskaltrust eigentlich?
+
+## Question
+
+Wo finde ich alle Infos?
+
+## Question
+
+Wo soll ich anfangen?
+
+## Metadata tags
+
+lang-de, market-all, middleware, PosCreator, POSDealer,Consultant
+
+## Answer
+
+1. fiskaltrust bietet Produkte und Dienstleistungen im Bereich [Compliance-as-a-Service](https://github.com/fiskaltrust/productdescription-de-doc/blob/master/compliance-as-a-service/overview.md) und [revisionssichere Daten-as-aService](https://github.com/fiskaltrust/productdescription-de-doc/blob/master/revisionssichere-daten-as-a-service/overview.md) für Kassensysteme an.
+2. Das Kernprodukt von fiskaltrust ist die [kostenfreie fiskaltrust.Middleware](https://github.com/fiskaltrust/productdescription-de-doc/blob/master/compliance-as-a-service/produkte/4445-0003-lokal-installierte-middleware.md) für [Kassenhersteller](https://github.com/fiskaltrust/productdescription-de-doc/blob/master/angebote-nach-kundensegment/Kassenhersteller.md). Damit können die wichtigsten Fiskalisierungsanforderungen in Deutschland umgesetzt werden. 
+3. Zusätzlich bietet fiskaltrust eine Vielzahl an [Addon-Produkte für Kassenhändler](https://github.com/fiskaltrust/productdescription-de-doc/blob/master/angebote-nach-kundensegment/Kassenhaendler.md). Diese Addon-Produkte können entweder einzeln, oder kostengünstig als [Sorglos-Pakete](https://github.com/fiskaltrust/productdescription-de-doc/tree/master/produkt-bundles) für All-in-One-Fiskalisierungslösungen gebündelt (z.B. Compliance Lösung + Revisionssichere Datenspeicher-Lösung + Technische Sicherheitseinrichtung aus einer Hand) für den Resale an Kassenbetreiber erworben werden.
+4. Information, Dokumentation zur Implementierung bzw. Verwendung, Beispiele und FAQ für fiskaltrust Produkte in allen Ländern findet man zentral auf [github](https://github.com/fiskaltrust).
```

Comment:
4. Information, Dokumentation zur Implementierung bzw. Verwendung, Beispiele und FAQ für fiskaltrust Produkte in allen Ländern findet man zentral [hier](https://docs.fiskaltrust.cloud) bzw. auf [github](https://github.com/fiskaltrust).

Reply:
done